toolkit/tools/imagecustomizer/README.md

```diff
@@ -46,7 +46,8 @@ Disadvantages:
 3. Install prerequisites: `qemu-img`, `rpm`, `dd`, `lsblk`, `losetup`, `sfdisk`,
    `udevadm`, `flock`, `blkid`, `openssl`, `sed`, `createrepo`, `mksquashfs`,
    `genisoimage`, `parted`, `mkfs`, `mkfs.ext4`, `mkfs.vfat`, `mkfs.xfs`, `fsck`,
-   `e2fsck`, `xfs_repair`, `resize2fs`, `zstd`, `veritysetup`.
+   `e2fsck`, `xfs_repair`, `resize2fs`, `tune2fs`, `xfs_admin`, `fatlabel`, `zstd`,
```

Comment:
should those be added in the dockerfile too?

Reply:
These tools are already included in the existing packages. Namely, `e2fsprogs`, `dosfstools`, and `xfsprogs`.